projects/js-packages/charts/src/components/pie-semi-circle-chart/stories/index.docs.mdx

```diff
@@ -0,0 +1,458 @@
+		width={ 600 }
+		clockwise={ false }
+		label="Counter-clockwise Rendering"
+	/>` }
+/>
+
+### Different Thickness Values
+
+Adjust the visual weight of the chart using the `thickness` prop:
+
+<Source
+	language="jsx"
+	code={ `// Thin ring (thickness: 0.2)
+		<PieSemiCircleChart
+			data={ data }
+			width={ 400 }
+			thickness={ 0.2 }
+			label="Thin Ring"
+		/>
+
+		// Thick ring (thickness: 0.8)
+		<PieSemiCircleChart
+			data={ data }
+			width={ 400 }
+			thickness={ 0.8 }
+			label="Thick Ring"
+		/>` }
+/>
+
+## Responsive Behavior
+
+### Responsive Width
+
+The chart can be made responsive by omitting the `width` prop, allowing it to adapt to its container:
+
+<Canvas of={ PieSemiCircleChartStories.Responsiveness } />
+
+<Source
+	language="jsx"
+	code={ `// Responsive chart adapts to container width
+	<PieSemiCircleChart
+		data={ data }
+		thickness={ 0.4 }
+		label="Responsive Chart"
+		withTooltips={ true }
+	/>` }
+/>
+
+The responsive behavior maintains the 2:1 aspect ratio (width:height) and scales proportionally.
+
+## Styling and Customization
+
+### Custom Colors
+
+Segments automatically use theme colors, but you can override individual segment colors using the `color` property in your data:
+
+<Source
+	language="jsx"
+	code={ `const customColorData = [
+		{
+			label: 'Primary',
+			value: 60,
+			percentage: 60,
+			color: '#3366CC', // Custom blue
+		},
+		{
+			label: 'Secondary',
+			value: 40,
+			percentage: 40,
+			color: '#DC3912', // Custom red
+		},
+	];
+
+	<PieSemiCircleChart
+		data={ customColorData }
+		width={ 400 }
+		label="Custom Colors"
+	/>` }
+/>
+
+### Label and Note Styling
+
+The chart includes built-in styling for labels and notes with appropriate typography hierarchy:
+
+- **Label**: 16px, font-weight 600, positioned above the chart
+- **Note**: 14px, positioned below the label
+
+<Source
+	language="jsx"
+	code={ `<PieSemiCircleChart
+		data={ data }
+		width={ 600 }
+		label="Primary Heading"
+		note="Secondary information or context"
+	/>` }
+/>
+
+### Theme Integration
+
+The chart automatically integrates with the charts theme system, using predefined colors, spacing, and typography. Theme colors are applied in sequence to chart segments based on their index in the data array.
+
+## Legend Integration
+
+### Legend Positioning
+
+Control legend placement using alignment properties:
+
+<Source
+	language="jsx"
+	code={ `// Legend at top-right
+	<PieSemiCircleChart
+		data={ data }
+		showLegend={ true }
+		legendAlignmentHorizontal="right"
+		legendAlignmentVertical="top"
+	/>
+
+	// Vertical legend on the right
+	<PieSemiCircleChart
+		data={ data }
+		showLegend={ true }
+		legendOrientation="vertical"
+		legendAlignmentHorizontal="right"
+	/>` }
+/>
+
+### Legend Shape Options
+
+Customize legend indicators with different shapes:
+
+<Source
+	language="jsx"
+	code={ `<PieSemiCircleChart
+		data={ data }
+		showLegend={ true }
+		legendShape="square" // Options: 'circle', 'square', 'line'
```

Comment:
Think this is supposed to be `'rect' | 'circle' | 'line'`

Reply:
Good catch. Fixed in daa23a20a0.